pkg/exprhelpers/exprlib.go

```diff
@@ -69,6 +70,8 @@ func GetExprEnv(ctx map[string]interface{}) map[string]interface{} {
 		"GetDecisionsSinceCount": GetDecisionsSinceCount,
 		"Sprintf":                fmt.Sprintf,
 		"ParseUnix":              ParseUnix,
+		"GetFromCache":           cache.GetKey,
+		"SetInCache":             cache.SetKey,
```

Comment:
is the term stash or cache?

Reply:
stop confronting me with my inconsistencies :sweat_smile: